spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java

```diff
@@ -38,14 +38,15 @@
 public class BusJacksonAutoConfiguration {
 
 	@Bean
-	@ConditionalOnMissingBean(name="busJsonConverter")
+	@ConditionalOnMissingBean(name = "busJsonConverter")
```

Comment:
I use the spring eclipse formatter. Think last commit includes the wrong formatter.